### DISCUSSION TITLE
Latest Update not working on RPI 3

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1015] [Fixed Issues]
I've big problems with lattest update on my RPI 3 .
In System the current is shown that it is below 4.7 V 
and in settings the player page did not launch.

**Comment by audio-drome** [comment]
Just installed 20220904 on RPI3b and settings page doesn't launch, so it's impossible to configure the player. 
No problems with voltage.

**Reply by rern** [reply]
Fixed by today's update.
